### PR TITLE
Update all static licenses

### LINKED ISSUE #589: Update all static licenses
1. Compare all license files
2. Check commits from the 2022 - now and update license files: https://github.com/ufal/clarin-dspace/commits/clarin/dspace-xmlui/src/main/webapp/themes/UFAL/lib/html

### PR DESCRIPTION
**Pull request by Paurikova2**
| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  3 |  0  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |
## Problem description
Missing static licenses. Problem with redirecting to licenses page.

### CODE REVIEW

**Comment by Paurikova2** [comment]
@milanmajchrak, because I set the licenses to no_static_licenses, the changes file ran through pre-commit (I think), and that is the reason why there are too many spaces being removed in some files

File: src/app/static-page/static-page.component.ts [hunk 1]
```diff
@@ -70,7 +70,9 @@ export class StaticPageComponent implements OnInit {
   private composeRedirectUrl(href: string | null, namespacePrefix: string): string {
     const staticPagePath = STATIC_PAGE_PATH;
     const baseUrl = new URL(window.location.origin);
-    baseUrl.pathname = `${namespacePrefix}/${staticPagePath}/`;
+    baseUrl.pathname = href.startsWith('no_static_')
```
Comment (milanmajchrak) [inline src/app/static-page/static-page.component.ts]: 'no_static' to constant

File: src/static-files/cs/faq.html [hunk 2]
```diff
@@ -35,7 +35,7 @@ <h3 id="what-is-the-repository">Co je repozitář?</h3>
 		<li>ukládání dat uživatelem s jistotou bezpečného uložení - všichni je mohou najít, používat i správně citovat (čímž uživatel získá příslušný kredit).</li>
 	</ul>
 	</div>
-	
+
```
Comment (milanmajchrak) [inline src/static-files/cs/faq.html:38]: Why? Please remove all unnecessary changes.

File: src/static-files/cs/item-lifecycle.html [hunk 3]
```diff
@@ -20,49 +20,49 @@ <h3 id="submitted-item">Nově vložené příspěvky</h3>
...
+      <p>Pokud není možné, obvykle z důvodu nejasného nebo omezeného právního statusu části dat, která ukládáte do repozitáře, přiřadit datům nebo nástroji otevřenou licenci, je možné dohodnout takovou <strong>restriktivní</strong> (omezenou) licenci, která bude vyžadovat explicitní souhlas uživatele s jejími podmínkami. Zdůrazňujeme, že toto nelze aplikovat na metadata položky - ta jsou vždy veřejně a otevřeně dostupná.
       Podporujeme tedy i takové restriktivní licence pro balíčky souborů s daty, které vyžadují před stahováním "elektronický podpis."
       Tyto elektronické podpisy uchováváme pro případy sporů o porušení práv k duševnímu vlastnictví.</p>
-      
-       <p><a href="licenses">Podívejte se na licence, které jsou v současné době k dispozici,</a> nebo nás <a class="helpdesk-tolink" href="#">kontaktujte</a>, pokud budete chtít nějakou speciální licenci přidat. 
+
+       <p><a href="no_static_licenses">Podívejte se na licence, které jsou v současné době k dispozici,</a> nebo nás <a class="helpdesk-tolink" href="#">kontaktujte</a>, pokud budete chtít nějakou speciální licenci přidat.
```
Comment (milanmajchrak) [inline src/static-files/cs/item-lifecycle.html:61]: This should be the only change in this file.

File: src/static-files/faq.html [hunk 4]
```diff
@@ -35,7 +35,7 @@ <h3 id="what-is-the-repository">What is the repository?</h3>
 			can find it, use it, and correctly cite it (giving you credit)</li>
 	</ul>
 	</div>
-	
+
```
Comment (milanmajchrak) [inline src/static-files/faq.html:38]: again, unnecessary change

File: src/static-files/cs/dummy.html [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<div id="faq-like">
```
Comment (milanmajchrak) [inline src/static-files/cs/dummy.html:1]: When is this `dummy.html` loaded? Is there logic which show this page if there is no translation?

**Review by milanmajchrak — CHANGES_REQUESTED** [review]

File: src/static-files/cs/dummy.html (hunk 5 above)
Comment (Paurikova2) [reply src/static-files/cs/dummy.html:1]: I did it based on> https://github.com/ufal/clarin-dspace/blob/d20cbdb420e7c7ac01bfea8e3285b4ffaff1c1ed/dspace-xmlui/src/main/webapp/themes/UFAL/lib/html/cs/dummy.html#L3

File: src/static-files/cs/faq.html (hunk 2 above)
Comment (Paurikova2) [reply src/static-files/cs/faq.html:38]: All of that made precommit I think

File: src/static-files/faq.html (hunk 4 above)
Comment (Paurikova2) [reply src/static-files/faq.html:38]: It ran pre-commit because I replaced 'license' with 'no_static_license', so this file was automatically checked. Should I change it?

**Review by milanmajchrak — DISMISSED** [review]

File: src/app/static-page/static-page.component.ts [hunk 6]
```diff
@@ -99,6 +102,9 @@ export class StaticPageComponent implements OnInit {
   }
 
   private redirectToAbsoluteLink(redirectUrl: string, href: string | null, namespacePrefix: string): void {
+    if (href.startsWith(StaticPageComponent.no_static)) {
+      href = href.replace(StaticPageComponent.no_static,'');
```
Comment (vidiecan) [inline src/app/static-page/static-page.component.ts]: TT
coding style

**Review by vidiecan — CHANGES_REQUESTED** [review]

**Review by vidiecan — APPROVED** [review]
